### ISSUE TITLE
Edits for child terms of GO:0005604 basement membrane 

### ISSUE THREAD

**Issue by rachhuntley** [geneontology/go-ontology#15652]
After consulting with Sylvie Ricard-Blum (MatrixDB) and based on https://www.ncbi.nlm.nih.gov/pubmed/8061357 it appears that the basement membrane is composed of a single layer made up of lamina densa material, rather than being a laminated structure consisting of several layers.

Could we make the following changes to these terms:

- Merge GO:0008003 lamina densa (0 annotations) with GO:0005604 basement membrane

- Obsolete GO:0008004 lamina reticularis: 0 annotations

- Obsolete GO:0008002 lamina lucida: 0 annotations 

Additionally, basal lamina is often used at a synonym of basement membrane (e.g. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3039528/)

Therefore: 

- Merge GO:0005605 basal lamina: (525 total annotations; 417 IEA) with GO:0005604 basement membrane.

Finally GO:0061618 sublamina densa has only one experimental annotation (BHF) from this paper http://europepmc.org/abstract/MED/15623520, in which the authors refer to the artefactual components above.  Therefore: 

- Obsolete GO:0061618 sublamina densa: (27 total annotations; 25 IEA)

See also #15300

**Comment by ukemi** [comment]
Notice sent May 1, 2018

Deal all, 

The proposal has been made to obsolete:
 GO:0008004 (lamina reticularis)- no annotations
GO:0008002 (lamina lucida)- no annotations
GO:0061618 (sublamina densa)- 27 total annotations; 25 IEA

The reason for these obsoletions is that it is believed that these structures are the result of experimental artefacts. 

We also plan to merge GO:0008003 (lamina densa, no annotations) and GO:0005605 (basal lamina, 525 total annotations; 417 IEA) into basement membrane.

A full explanation of the rationale for these changes can be found in this issue:
https://github.com/geneontology/go-ontology/issues/15652

Please add comments directly to this issue. 
We are opening a comment period for these proposed changes. We'd like to proceed and obsolete this term on May 14, 2018. Unless objections are received by  May 14, 2018  we will assume that you agree to this change.

From ontobee it appears that some of these classes may also be used in the mammalian phenotype ontology, the human phenotype ontology, and the behavior ontology.

Thanks,

David & Rachael

**Comment by ukemi** [comment]
Obsoletions done in PR  https://github.com/geneontology/go-ontology/pull/15889

**Comment by cmungall** [comment]
I realize the PR has merged but I'm just conferring with some expert anatomists, they say "I have no problem with lamina lucida and lamina densa being considered artifacts, but that lamina basalis and lamina reticularis should probably stay as important parts of basement membranes", I'm just requesting more info...

**Comment by rachhuntley** [comment]
Hi Chris, We had based these changes on correspondence with Sylvie Ricard-Blum, an expert on ECM who built MatrixDB. I've pasted the relevant parts of our correspondence below. Specifically, basal lamina is considered a synonym of basement membrane, and the quote at the end of this comment "the lamina lucida is an artefact formed during conventional tissue preparation, and in its original condition in the living state, the basement membrane is composed of a single layer made up of lamina densa material", which led us to decide that only lamina densa and basement membrane are authentic. Note that most of the obsoleted/merged terms had no annotations.

But you can see from my questions that this is a confusing area, that's why we decided to trust Sylvie's expert opinion.

My question: 
Is the basal lamina a part of the basement membrane or a type of basement membrane? (Wikipedia has it as part of https://en.wikipedia.org/wiki/Basal_lamina)

Response from Sylvie:
In the extracellular matrix field basal lamina is often used at a synonym of basement membrane.
- See Yurchenco, 2011 (first line, first paragraph)
- See references in Basal lamina an overview ScienceDirect Topics

Also, our correspondence regarding child terms of basement membrane:
Le 17/04/2018 à 10:55, Huntley, Rachael a écrit :
Dear Sylvie,

Thank you so much for this, it is very useful.

I’m still confused by the basal lamina though. There seem to be some references using basal lamina as a synonym of basement membrane, but then others are saying that this is not correct, e.g.  the wikipedia entry for basal lamina refers to the differences between BL and BM (https://en.wikipedia.org/wiki/Basal_lamina), i.e. The BM contains the lamina lucida, lamina densa and lamina reticularis, but the BL contains only the first two.

Then there is this reference: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3299390/, which says "The basement membrane underlies the typical cellular epithelium or vascular endothelium and consists of two thin structural layers. The first layer consists of a basal lamina that is synthesized by epithelial or endothelial cells that differ in their respective characteristics [96]. The second layer is the reticular lamina made by fibroblasts, among other surrounding cells [97].” Figure 3.

The GO currently has this structure:
basement membrane
- basal lamina (part_of)
-- lamina densa (part_of)
--lamina reticularis (part_of)
--lamina lucida (part_of)

So, based on the information above, I was considering re-organising this to:

basement membrane
-lamina reticularis (part_of)
-basal lamina (part_of)
--lamina densa (part_of)
--lamina lucida (part_of)

Dear Rachel, 
i would not trust Wikipedia content, which is not always reviewed, or the article entitled "Prostaglandins in Cancer Cell Adhesion, Migration, and Invasion" (PMC3299390), which was not written by experts in the field of basement membranes. 
I attach to this e-mail a paper entitled "Lamina lucida of basement membrane: an artefact" (PMID:8061357), which concludes that that "the lamina lucida is an artefact formed during conventional tissue preparation, and in its original condition in the living state, the basement membrane is composed of a single layer made up of lamina densa material".

Sylvie.

Professor Sylvie RICARD-BLUM
Pericellular and Extracellular Supramolecular Assemblies
UMR 5246 CNRS - University Lyon 1
Institute of Molecular and Supramolecular Chemistry and Biochemistry
University Claude Bernard Lyon 1, 43 Boulevard du 11 novembre 1918
69622 Villeurbanne Cedex, France
Phone: +33(0)4 72 44 82 32
E-mail: sylvie.ricard-blum@univ-lyon1.fr
http://www.icbms.fr
The French Society for Matrix Biology http://www.sfbmec.fr
The International Society for Matrix Biology http://ismb.org

**Comment by cmungall** [comment]
OK great I think our experts are in agreement! I'm confused therefore with the obsoletion of the lamina reticularis. Is the intent to reinstate it with a new ID?

**Comment by ukemi** [comment]
Hi @rachhuntley . Should we create a new lamina reticularis with a new def?

**Comment by rachhuntley** [comment]
Hi @ukemi @cmungall 
From this quote "the basement membrane is composed of a single layer made up of lamina densa material", and Sylvie's mistrust of the article I had mentioned, I had inferred that the other components were artefacts. I would like to double-check with Sylvie whether she thinks the lamina reticularis is real, and if so, what it is composed of. There are several references on the subject but, as we've found, it's difficult to see what is genuine. If we were to reinstate the term, we need an accurate definition and a trustworthy reference.

**Comment by ukemi** [comment]
I agree @rachhuntley. And in that case, I think that rather than 'unobsolete' the previous term, it would make more sense to add a new term with the same string and provide a clear definition. I must admit, the part of relationship wouldn't make sense to me if there are no other parts.

**Comment by rachhuntley** [comment]
I'm still awaiting feedback from Sylvie regarding the lamina reticularis term, but I have heard back from two other basement membrane experts, whose responses I've copied below.

To summarise, they consider that the basement membrane laminations are likely artefacts of the fixing method that was traditionally used and would not recommend the use of a lamia reticularis term (or lamina lucida, sublamina densa). From recent ultra high resolution optical microscopy and EM studies (that they both cite) there does appear to be some partitions, but it's not clear this is the same for all basement membranes. There is still a lot to learn about basement membranes and how they differ in different cells, tissues and organisms, therefore it's probably best to stick with just the basement membrane term.

I think we should add a comment to the basement membrane term (and maybe the obsolete terms too?) citing the 3 papers in their responses, in case this comes up again.
-------
Peter Yurchenco and Rachel Lennon's replies in response to my question "Do you think lamina reticularis is a genuine component?"

Dear Dr. Huntley,
I think you raise an interesting issue. As your attached reference pointed out, rapid freezing/freeze substitution strongly suggested that the basement membrane laminations seen by conventionally fixed and stained tissues (notably the GBM) are artifacts of sample preparation. I have no data to argue otherwise and found the old concept difficult to explain by the known basement membrane binding interactions, shapes and sizes. I think one of the most revealing studies in recent years is an ultra high resolution optical microscopy (STORM) and EM study of the GBM from the lab of Jeff Miner (PMID:24137544). The study argues that the GBM, which is actually a fusion of two basement membranes, has a cross-sectional organization reflecting the fusion and not one corresponding to the old conventional EM images. I agree the concept of a lamina reticularis is outdated. A problem in banning the term is that there is a large body of published studies from years gone by that uses the term.
Sincerely,
Peter
------------
Peter D. Yurchenco M.D., Ph.D.
Professor of Pathology & Laboratory Medicine
Rutgers Univ. - Robert W. Johnson Medical School
------

Dear Rachael,
Thanks for your interesting question-
Personally, I do not use the terms 'lamina reticularis', 'lamina lucida' or 'sublamina densa' but instead I use the term basement membrane to refer to the whole structure.
 
There is some evidence of component partitioning in BMs and I attach an eLife paper from Jeff Miner’s group (PMID:24137544) describing the nanoscale architecture of the glomerular BM. Such partitioning may give rise to different ultrastructural appearances but I’m not sure it helps to give these partitions different names until we know whether they apply widely to other BMs.
 
There are also instances of two BMs adhering during development in some tissues and the composition/ultrastructure of these BMs may well be different. I attach a recent review on this subject from David Sherwood’s group (PMID:29803937).
 
Finally, we know the protein complexity BMs is significant- whilst the most abundant components are laminins (Peter’s expertise) and type IV collagen networks, there are lower levels of many other proteins that may have an influence on the ultrastructure that we don’t yet understand. I attach a review from my group on the use of mass spectrometry to try and capture this complexity (PMID: 27553508).
 
In short- there is much to learn about basement membranes but for now I would consider the structure as a whole unit. I would be very interested to hear Peter’s perspective!
 
With best wishes,
Rachel
 
Professor Rachel Lennon 
Wellcome Trust Senior Research Fellow in Clinical Science   I  Lead for Integrated Clinical Academic Training  I  Honorary Consultant Paediatric Nephrologist   I  Wellcome Trust Centre for Cell-Matrix Research   I  Division of Cell Matrix Biology and Regenerative Medicine   I  School of Biological Sciences  I  Faculty of Biology Medicine and Health  I  The University of Manchester  I  Room C.3257, Michael Smith Building  I  Oxford Road I Manchester M13 9PT

**Comment by cmungall** [comment]
Relaying comments from Paul Neumann, FIPAT. We're still waiting to hear from the FIPAT histology group:


My immediate response would be that I have no problem with lamina lucida and lamina densa being considered artifacts, but that lamina basalis and lamina reticularis should probably stay as important parts of basement membranes (membranae basales).  


There are two types of basement membranes: 

Typical: composed of lamina basalis and lamina reticularis, where epithelium sits on connective tissues.

Special: composed of two laminae basales (e.g., alveolar walls of lung, glomeruli of kidneys, and placenta), where two epithelia are almost directly opposed, permitting greater diffusion of fluid, solutes and gases.    


The lamina basalis is considered at least partially a product of epithelial cells, whereas the lamina reticularis is considered to be largely a product of connective tissue cells (e.g., type 3 collagen). 


The issue about three layers of plasmalemma is a different issue entirely.  The lipid bilayer is sometimes described as having an outer and inner hydrophilic zones (heads of phosopholipids) and an intermediate hydrophobic zone (tails of phospholipids).  So the concept of three zones is a good one, even if it doesn't match up precisely with the EM images.

**Comment by ukemi** [comment]
@rachhuntley has begun a discussion with the relevant experts about the fate of the lamina reticularis term. As agreed at the ontology editors' meeting on July 2, we resurrected this term.
